server/claim/claim.service.ts

```diff
@@ -77,7 +77,7 @@ export class ClaimService {
                 TargetModel.Claim,
                 user,
                 HistoryType.Create,
-                newClaim.latestRevision
```

Comment:
If we do that we start to duplicate data unnecessarily, why do we need to do that?

Reply:
As discussed in slack, let's keep the revision id as the `before` and `after` content and when requesting the specific history entry, we will do an extra query to pull the content from the revisions cc/ @ThaisMap